Formula/ftjam.rb

```diff
@@ -24,11 +24,24 @@ class Ftjam < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "03bc2f284df267e0e2f669480b142ae11c5bf15066eebc0dc7de69c42e116d41"
   end
 
+  deprecate! date: "2022-09-18", because: :unmaintained
```

Comment:
Last release was from 2006.

Freetype dropped Jam build system in 2.10.2 (2020) so doesn't seem likely they will keep maintaining `ftjam`.

Debian 12+ and Ubuntu 22.10+ have dropped package.